### PR TITLE
Add dnsConfig in values.yaml

### PR DESCRIPTION
**Pull request by bcdady**
Address DNS issues in #73

### CODE REVIEW

File: cloudhealth-collector/values.yaml [hunk 1]
```diff
@@ -7,10 +7,18 @@
...
 replicaCount: 1
 
+# Required values, to be specified per deployment:
 apiToken: ""
 clusterName: ""
 
+# The remaining are default values
 collectionIntervalSecs: 900
+
+dnsConfig:
```
Comment (gm-cht) [inline cloudhealth-collector/values.yaml:17]: The default has to be an empty `{}`. The `deployment.yaml` file has to change to include this option parameter.

**Review by gm-cht — CHANGES_REQUESTED** [review]

File: cloudhealth-collector/values.yaml (hunk 1 above)
Comment (bcdady) [reply cloudhealth-collector/values.yaml:17]: I'll look into adding this soon.

Comment (bcdady) [reply cloudhealth-collector/values.yaml]: Hi @gm-cht Could you re-review and let me know if you see anything else that needs follow-up here?

File: cloudhealth-collector-image-docs/CHANGELOG.md [hunk 2]
```diff
@@ -9,65 +9,88 @@ The agent has been verified against:
...
+
 ### Removed
+
 * Removed jbundler
 
 ## [1119] - 2022-03-16
-### Added
-* Changelog introduced
 
+### Added
```
Comment (pjiten) [inline cloudhealth-collector-image-docs/CHANGELOG.md:95]: Everything looks good to me. I am just curious if changes in this file CHANGELOG.md are unintentional, caused by an editor or something.

**Review by gm-cht — APPROVED** [review]

File: cloudhealth-collector/values.yaml (hunk 1 above)
Comment (gm-cht) [reply cloudhealth-collector/values.yaml]: @bcdady looks good to me. let us run some tests our end and will merge this.

File: cloudhealth-collector-image-docs/CHANGELOG.md (hunk 2 above)
Comment (bcdady) [reply cloudhealth-collector-image-docs/CHANGELOG.md:95]: Thanks @pjiten. Yes the markdown edits were to make markdownlint happy.

Comment (pjiten) [reply cloudhealth-collector-image-docs/CHANGELOG.md:95]: Awesome. I am performing some usual testing in local sandbox. I will approve as soon as I finish it today.

**Review by pjiten — APPROVED** [review]

**Review by wxia20 — APPROVED** [review]

**Pull request merged**